.github/workflows/test.yml

```diff
@@ -31,3 +31,42 @@ jobs:
         run: yarn build
       - name: Run unit tests
         run: yarn test
+  build:
```

Comment:
Which step uploads dockerfile to docker hub?

Reply:
currently there is no dockerhub upload: https://github.com/leocode/image-service/pull/13#issuecomment-893220875

Reply:
I will add build and upload and docker hub side directly, here we only test if image builds at all